src/commands/run.rs

```diff
@@ -832,18 +832,33 @@ fn generate_coredump(err: &anyhow::Error, source_name: &str, coredump_path: &str
         .ok_or_else(|| anyhow!("no wasm backtrace found to generate coredump with"))?;
 
     let coredump = wasm_encoder::CoreDumpSection::new(source_name);
+    let mut coremodules = wasm_encoder::CoreDumpModulesSection::new();
     let mut stacksection = wasm_encoder::CoreDumpStackSection::new("main");
+    let mut modulenames = std::collections::HashSet::new();
     for f in bt.frames() {
+        modulenames.insert(f.module_name().unwrap_or("<unknown>"));
```

Comment:
Should the `modulenames` set be deferred until the frame -> instance -> module mapping can be created? Otherwise there isn't anything pointing to the module names I think?